crates/driver/src/infra/solver/dto/solution.rs

```diff
@@ -373,9 +375,13 @@ enum SigningScheme {
     Eip1271,
 }
 
+#[serde_as]
 #[derive(Debug, Deserialize)]
 #[serde(rename_all = "lowercase", deny_unknown_fields)]
 pub enum Score {
-    Solver(eth::U256),
+    Solver {
+        #[serde_as(as = "serialize::U256")]
```

Comment:
This should probably use `HexOrDecimalU256` so that solvers can encode a score in a more normal format as well.

Reply:
`HexOrDecimalU256` is used in legacy system. For colocated driver we always use `serialize::U256`. 

If you prefer using `HexOrDecimalU256` as more future proof, let's than substitute all occurrences in a separate PR.

Reply:
Oh right. Given that we have our own type now (eth::U256), can't we implement serialize on this type so that we no longer forget adding this custom `serde_as` to all the types?

Reply:
> Given that we have our own type now (eth::U256), can't we implement serialize on this type so that we no longer forget adding this custom serde_as to all the types?

`eth::U256` is just a reexported type so we can't implement traits for it. We'd have to create a completely new type (or a wrapper). That would probably result in a HUGE diff.

Reply:
https://github.com/cowprotocol/services/issues/2086